v20.1/temporary-tables.md

```diff
@@ -0,0 +1,211 @@
+
+<div>
+  {% include {{ page.version.version }}/sql/diagrams/opt_temp_create_table.html %}
+</div>
+
+
+## Example
+
+To turn on temp tables:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SET experimental_enable_temp_tables=on;
+~~~
+
+To create a temp table:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE TEMP TABLE users (
+        id UUID PRIMARY KEY,
+        city STRING,
+        name STRING,
+        address STRING
+);
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW CREATE TABLE users;
+~~~
+
+~~~
+  table_name |                create_statement
+-------------+-------------------------------------------------
+  users      | CREATE TEMP TABLE users (
+             |     id UUID NOT NULL,
+             |     city STRING NULL,
+             |     name STRING NULL,
+             |     address STRING NULL,
+             |     CONSTRAINT "primary" PRIMARY KEY (id ASC),
+             |     FAMILY "primary" (id, city, name, address)
+             | )
+(1 row)
+~~~
+
+To show the newly created `pg_temp` schema, use [`SHOW SCHEMAS`](show-schemas.html):
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW SCHEMAS;
+~~~
+
+~~~
+           schema_name
+---------------------------------
+  crdb_internal
+  information_schema
+  pg_catalog
+  pg_temp_1584540651942455000_1
+  public
+~~~
+
+
+To create another temp table that references `users`:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE TEMP TABLE vehicles (
+        id UUID NOT NULL,
+        city STRING NOT NULL,
+        type STRING,
+        owner_id UUID,
+        creation_time TIMESTAMP,
+        CONSTRAINT "primary" PRIMARY KEY (city ASC, id ASC),
+        INDEX vehicles_auto_index_fk_city_ref_users (city ASC, owner_id ASC)
+);
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW CREATE TABLE vehicles;
+~~~
+
+~~~
+  table_name |                             create_statement
+-------------+----------------------------------------------------------------------------
+  vehicles   | CREATE TEMP TABLE vehicles (
+             |     id UUID NOT NULL,
+             |     city STRING NOT NULL,
+             |     type STRING NULL,
+             |     owner_id UUID NULL,
+             |     creation_time TIMESTAMP NULL,
+             |     CONSTRAINT "primary" PRIMARY KEY (city ASC, id ASC),
+             |     INDEX vehicles_auto_index_fk_city_ref_users (city ASC, owner_id ASC),
+             |     FAMILY "primary" (id, city, type, owner_id, creation_time)
+             | )
+(1 row)
+~~~
+
+To show all temp tables in a session's temporary schema, use `SHOW TABLES FROM pg_temp`:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW TABLES FROM pg_temp;
+~~~
+
+~~~
+  table_name
+--------------
+  users
+  vehicles
+(2 rows)
+~~~
+
+You can also use the full name of the temporary schema in the `SHOW` statement (e.g., `SHOW TABLES FROM pg_temp_1584540651942455000_1`).
+
+{{site.data.alerts.callout_info}}
```

Comment:
This feels slightly less example-y and more about-y, but maybe it's a subhead called "Visibility of temporary tables" or similar?